components/org.wso2.carbon.identity.conditional.auth.functions.elk/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/elk/util/ElasticConfigProvider.java

```diff
@@ -0,0 +1,79 @@
+package org.wso2.carbon.identity.conditional.auth.functions.elk.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Map;
+
+public class ElasticConfigProvider {
```

Comment:
Add the class description.

Reply:
Resolved with [f8c134b](https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/98/commits/f8c134b5cf1651ab35b0f4d346b4a71ea711c50a)